pgml-extension/src/lib.rs

```diff
@@ -24,6 +24,7 @@ extension_sql_file!("../sql/schema.sql", name = "schema");
 #[cfg(not(feature = "use_as_lib"))]
 #[pg_guard]
 pub extern "C" fn _PG_init() {
+    bindings::python::activate().expect("Error setting python venv");
```

Comment:
This works? I ask because each connection forks from the postmaster. It gets the venv set correctly? :O

Reply:
It works for me locally. I can test from multiple different connections.

Reply:
Can confirm tested from three different connections and it worked

Reply:
Well, that was easy.